I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 30, in script
    show alert onlayer screen at mid
TypeError: 'tuple' object is not callable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 30, in script
    show alert onlayer screen at mid
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\ast.py", line 1131, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\ast.py", line 1101, in show_imspec
    atl=atl)
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\exports.py", line 691, in show
    img = i(img)
TypeError: 'tuple' object is not callable

Windows-8-6.2.9200
Ren'Py 7.0.0.196
Escape Roomba 0.0
Tue Sep 18 21:39:17 2018
